core/test/qa_Messages.cpp

```diff
@@ -112,7 +112,7 @@ const boost::ut::suite MessagesTests = [] {
 
         "Block<T>-level heartbeat tests"_test = [] {
             gr::MsgPortOut toBlock;
-            TestBlock<int> unitTestBlock({ { "name", "UnitTestBlock" } });
+            TestBlock<int> unitTestBlock(property_map{ { "name", "UnitTestBlock" } });
```

Comment:
Should this be now preferable method for settings initialization, or did you change it just to test the new Block constructor?

Reply:
Should not be strictly needed. In the end I left the `std::initializer_list`-based constructor that delegates to the `property_map` based one.